Neo4j.Driver/Neo4j.Driver/Internal/Connector/TcpSocketClient.cs

```diff
@@ -201,9 +201,9 @@ internal void ConnectSocket(IPAddress address, int port)
                     }
                     catch (Exception e)
                     {
-                        _logger?.Error($"Failed to close connect to the server {address}:{port}" +
-                                       $" after connection timed out {_connectionTimeout.TotalMilliseconds}ms" +
-                                       $" due to error: {e.Message}.", e);
+                        _logger?.Error(e, $"Failed to close connect to the server {address}:{port}" +
+                                          $" after connection timed out {_connectionTimeout.TotalMilliseconds}ms" +
+                                          $" due to error: {e.Message}.");
```

Comment:
Won't message get duplicated here because it is included in the message string and exception contains it? Same for the next `_logger?.Error` call